docs/build_awx_image.md

```diff
@@ -0,0 +1,30 @@
+# Building the AWX Image
+
+## Build & Push Image
+
+To build and use a custom awx image with the awx-operator, use the `build_image` role:
+
+```
+$ ansible-playbook tools/ansible/build.yml -v -e @vars.yml
```

Comment:
It might be better to write out something like:

```
-e awx_image=registry.example.com/awx -e awx_version=test
```

Reply:
whoops, thanks.  that is fixed now.